aries-site/src/pages/templates/global-header.mdx

```diff
@@ -1,73 +1,127 @@
-  horizontalLayout
-  height="none"
-  width={{ min: 'medium' }}
->
-  <AppIdentityExample />
-</Example>
+Most sites should choose the default which will be the HPE logo for regular mode or the GreenLake logo for
+`console` mode. You can force it to use one or the other by specifically choosing HPE versus GreenLake if
+needed.
 
-### Header Nav
+![HPE logo](/templateImages/globalheader-hpe.png)
 
-The header navigation items should only be present if a user is logged in. The content of this section is flexible but should follow the guidelines established by [Header](/components/header).
+![GreenLake logo](/templateImages/globalheader-greenlake.png)
 
-The **Avatar button** opens additional options regarding the active account.
+### Header Main Navigation
 
-- If a user has a profile picture, this should appear in the avatar. Otherwise, their first and last initials should appear.
-- On click, a drop should open containing the user's name and email. The footer of the drop contains a "My Profile" button which routes to the user's profile page and a "Sign Out" button which signs the user out and routes back to the Sign In page.
+The main navigation in the center of the header provide marketing content. The links allow users to find
+HPE products, contact support and get more information about HPE. These should also typically be enabled
+for most sites.
 
-On mobile, the Avatar should remain in the header, but the remaining nav items should be moved into the sidebar menu.
+![Global Header Main Navigation example](/templateImages/globalheader-nav.png)
 
-<Example
-  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/extend/global-header/HeaderNav.js"
-  horizontalLayout
-  height="none"
->
-  <HeaderNavItems />
-</Example>
+### Search
+
+The *Search* icon allows the user to search hpe.com. 
+
+<Search />
+
+### Shopping Cart
+
+The *Shopping Cart* icon goes to the HPE shopping cart for buy.hpe.com. This should only
+be enabled for sites which have products or solutions for sale.
+
+<Shop />
+
+### Country Selector
+
+The country selector allows the user to go to a language-specific version of hpe.com. This
+should be disabled for most sites since the link really only takes you back to hpe.com
+instead of staying on your site.
+
+<Language />
+
+### Sign in
+
+Sign in allows the user to sign in to their HPE Passport account for hpe.com. Enable this
+option if your site uses features that require a Passport login.
+
+![Sign In button](/templateImages/globalheader-sign-in.png)
+
+### Cube Menu
+
+The *Cube Menu* allows access to HPE services available to the user such as GreenLake and
+other Cloud console applications as well as general HPE links for HPE content like
+developer.hpe.com, education.hpe.com, etc. This menu should be enabled to allow access
+to other HPE sites and applications.
+
+<AppsRounded />
+
```

Comment:
Include Color Scheme option?

Reply:
I wasn't sure what to do with that option. I think I'll check if there's some guidance we can provide for this choice.

Reply:
This is the most documentation I've seen and it doesn't say much.
[common-hfws-quick-js-guide-1.0.0 (3).pdf](https://github.com/grommet/hpe-design-system/files/6593348/common-hfws-quick-js-guide-1.0.0.3.pdf)

What about:
"Color Scheme allows implementors to choose either a light or dark variant of the header/footer. The color scheme you choose should be harmonious with the primary background of your application's header and avoid stark contrast which interrupts a user's visual flow. If your application supports light and dark modes, consider setting the `color_scheme` parameter with a value of either `light` or `dark` in alignment with the user's preference."

To the last point, in our implementation of Global Header, we could reference the localStorage variable `darkMode` --> there is a `useDarkMode()` hook in utils.

Reply:
Thanks for the wording. I think that works. I like the useDarkMode hook but I don't think we should recommend using an undocumented grommet API.

Reply:
> I don't think we should recommend using an undocumented grommet API.

Agree. Was meaning that on the Design System site we should change the Global Header color_scheme to match the themeMode someone has chosen for the site.

Reply:
That would be an enhancement outside of this PR, it just happened to be an aside thought I had as I was writing the guidance for the page.